docker-compose.yml

```diff
@@ -61,6 +61,25 @@ services:
       - GOST_API_URL=http://app:3000/
     env_file:
       - ./packages/client/.env
+  
+  nextjs:
+    container_name: gost-nextjs
```

Comment:
How does nextjs play into it?  I am not super familiar.

Reply:
This is a good point - I've written up a separate ticket for an ADR around using Nextjs [here](https://github.com/usdigitalresponse/usdr-gost/issues/1310) - it's one of several possible react frameworks we could use. I give a bit of reasoning there why one might consider using a framework such as this, but I'm realizing how exactly to accomplish this is itself a separate decision!